packages/tools/generator-widget/generators/app/templates/commons/README.md.ejs

```diff
@@ -14,4 +14,11 @@
 [link to GitHub issues]
 
 ## Development and contribution
-[specify contribute]
+
+1. Install NPM package dependencies by using: `npm install`. If you use NPM v7.x.x, which can be checked by executing `npm -v`, execute: `npm install --legacy-peer-deps`.
+1. Run `npm start` to watch for code changes. On every change:
+    - the widget will be bundled;
+    - the bundle will be included in a `dist` folder in the root directory of the project;
+    - the bundle will be included in the `deployment` and `widgets` folder of the Mendix test project.
```

Comment:
I'm a bit confused. Is this the README template that will be added with the generator when creating new widgets? I can understand adding the first line about NPM >= v7, but why are we suddenly adding this `npm start` part?

Reply:
To give the section a bit more sense. Just having that note about NPM 7 feels like an incomplete section.